src/main/java/spim/io/OMETIFFHandler.java

```diff
@@ -54,7 +54,10 @@ public OMETIFFHandler(CMMCore iCore, File outDir, String xyDev,
 		deltat = iDeltaT;
 		outputDirectory = outDir;
 		this.acqRows = acqRows;
-
+		tiles = tileCount;
+		int angles = Math.floorDiv(stacks, tiles);
```

Comment:
I agree. And in this case, I think a simple `(int) (stacks / tiles)` should be enough (because we do not expect negative values here).